ocean_lib/ocean/ocean.py

```diff
@@ -203,6 +206,33 @@ def get_nft_factory(self, nft_factory_address: str = "") -> ERC721FactoryContrac
 
         return ERC721FactoryContract(self.web3, nft_factory_address)
 
+    # Temporary solution until FixedRateExchange class will be fixed
+    @enforce_types
+    def create(self, data_token: str, exchange_rate: int, wallet: Wallet) -> str:
+        assert exchange_rate > 0, "Invalid exchange rate, must be > 0"
+        exchange = self._exchange_contract()
+        tx_id = exchange.create(
+            self.ocean_address, data_token, exchange_rate, from_wallet=wallet
```

Comment:
actually, please check the pool creation for uniformity. If I retrieve ocean address there instead of using a base token, then let's do it like that for conformity. If I pass in the ocean token, then do that for conformity :D